app/build.gradle

```diff
@@ -250,4 +250,7 @@ dependencies {
     // Third party libraries
     implementation 'com.github.a914-gowtham:compose-ratingbar:1.3.4'
     implementation 'com.github.akshaaatt:Logger-Android:1.0.0'
+
+    //Charting Library (Vico)
+    implementation("com.patrykandpatrick.vico:compose:1.13.1")
```

Comment:
Can we please give an attribution to this library on the settings page? I think we already attribute the animations and icons we use there